packages/web-client/app/utils/web3-strategies/layer1-chain.ts

```diff
@@ -264,15 +264,22 @@ export default abstract class Layer1ChainWeb3Strategy
 
   async refreshBalances() {
     if (!this.isConnected) return;
-    let balances = await Promise.all<string>([
-      this.getDefaultTokenBalance(),
-      this.getErc20Balance('DAI'),
-      this.getErc20Balance('CARD'),
-    ]);
-    let [defaultTokenBalance, daiBalance, cardBalance] = balances;
-    this.defaultTokenBalance = new BN(defaultTokenBalance);
-    this.daiBalance = new BN(daiBalance);
-    this.cardBalance = new BN(cardBalance);
+
+    try {
+      let balances = await Promise.all<string>([
+        this.getDefaultTokenBalance(),
+        this.getErc20Balance('DAI'),
+        this.getErc20Balance('CARD'),
+      ]);
+      let [defaultTokenBalance, daiBalance, cardBalance] = balances;
+      this.defaultTokenBalance = new BN(defaultTokenBalance);
+      this.daiBalance = new BN(daiBalance);
+      this.cardBalance = new BN(cardBalance);
+    } catch (e) {
+      if (!e.message.includes('what name the network id')) {
```

Comment:
I think it would be helpful to add an explanation comment here and including the whole error message in it.

Reply:
agreed, done, thank you!